libraries/src/Form/FormRule.php

```diff
@@ -20,6 +23,8 @@
 	 *
 	 * @var    boolean
 	 * @since  1.6
+	 *
+	 * @deprecated 4.2 Will be removed without replacement in Joomla 5.0 (Also remove phpcs exception)
```

Comment:
First comes the target version where it got removed

```suggestion
	 * @deprecated 5.0 Will be removed without replacement (Also remove phpcs exception)
```